.github/workflows/push_openapi_spec_to_readme.yml

```diff
@@ -17,7 +24,7 @@ jobs:
       - name: Load secrets
         uses: aws-actions/aws-secretsmanager-get-secrets@v2
         with:
-          secret-ids: README_API,beta/github_actions_push_openapi_spec_to_readme/readme_api_key
+          secret-ids: README_API,prod/github_actions_push_openapi_spec_to_readme/readme_api_key
```

Comment:
Updating this to `prod` based off [this comment](https://github.com/Asana/openapi/pull/77#discussion_r1560120151)